cluster/saltbase/salt/docker/docker.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=network.target docker.socket
+Requires=docker.socket
+
+[Service]
+EnvironmentFile={{ environment_file }}
+ExecStart=/usr/bin/docker -d -H fd:// "$DOCKER_OPTS"
+MountFlags=slave
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=infinity
+Restart=always
+RestartSec=2s
+StartLimitInterval=0
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I think that's a good idea; definitely at this stage we want to minimizing changes!

I think this whole thing is in the `else` block of `{% if grains.os_family == 'RedHat' %}`, so I think we get the (more complete) docker service as provided by the OS on RedHat, and we only have to install our own on non-Redhat systemd (i.e. Jessie and Vervet).